Firewall Top Flow Log Add Links (#126751)
Azure/azure-content: articles/firewall/monitor-firewall.md

- [Application rule aggregation log](/azure/azure-monitor/reference/tables/azfwapplicationruleaggregation) - Contains aggregated Application rule log data for Policy Analytics.
- [Network rule aggregation log](/azure/azure-monitor/reference/tables/azfwnetworkruleaggregation) - Contains aggregated Network rule log data for Policy Analytics.
- [NAT rule aggregation log](/azure/azure-monitor/reference/tables/azfwnatruleaggregation) - Contains aggregated NAT rule log data for Policy Analytics.

> - [Top flow log](/azure/azure-monitor/reference/tables/azfwfatflow) - The Top Flows (Fat Flows) log shows the top connections that are contributing to the highest throughput through the firewall. [Additional Documentation](https://learn.microsoft.com/en-us/azure/firewall/monitor-firewall-reference#top-flows)

Reviewer: ```suggestion
- [Top flow log](/azure/azure-monitor/reference/tables/azfwfatflow) - The Top Flows (Fat Flows) log shows the top connections that are contributing to the highest throughput through the firewall. For more information, see [Top flows log](monitor-firewall-reference.md#top-flows).
```

Once you configure the Log Analytics workspace, you can enable structured logs in Azure Firewall by navigating to the Firewall's **Diagnostic settings** page in the Azure portal. From there, you must select the **Resource specific** destination table and select the type of events you want to log.

> [!NOTE]

> > * There's no requirement to enable this feature with a feature flag or Azure PowerShell commands.

Reviewer: ```suggestion
```
This line isn't necessary anymore and it contradicts the new line we added.

> > <BR><BR> To enable Azure Firewall Fat Flow Log it requires [additional steps](https://learn.microsoft.com/en-us/azure/firewall/monitor-firewall-reference#top-flows).

Reviewer: ```suggestion
> * To enable Azure Firewall Fat Flow Log (Top flow log) you need to configure it through Azure PowerShell. For more information, see [Top flows log](monitor-firewall-reference.md#top-flows).
```

> [!NOTE]

> > There's no requirement to enable this feature with a feature flag or Azure PowerShell commands.

Reviewer: ```suggestion
> * There's no requirement to enable this feature with a feature flag or Azure PowerShell commands.
```
